docs/beta/tutorial/get_version.rst

```diff
@@ -0,0 +1,82 @@
+Tutorial: Get Tamr version
+==========================
+This tutorial will cover basic Python client usage by guiding you through:
+
+#. Configuring the connection to a Tamr instance
+
+#. Retrieving the version of that instance
+
+Prerequisites
+-------------
+To complete this tutorial you will need:
+
+- ``tamr-unify-client`` :doc:`installed <../../user-guide/installation>`
+
+- access to a Tamr instance, specifically:
+
+  - a username and password that allow you to log in to Tamr
+
+  - the socket address of the instance
+
+The socket address is composed of
+
+#. The protocol, such as ``"https"`` or ``"http"``
+
+#. The host, which may be ``"localhost"`` if the instance is deployed from the same machine from which your Python code will be run
+
+#. The port at which you access the Tamr user interface, typically ``9100``
+
+When you view the Tamr user interface in a browser, the url is ``<protocol>://<host>:<port>``. If the port is missing, the URL is simply ``<protocol>://host``.
+
+Steps
+-----
+The Session
+~~~~~~~~~~~
+The Tamr Python client uses a ``Session`` to persist the user's authentication details across requests made to the server where Tamr is hosted.
+
+A ``Session`` carries authentication credentials derived from a username and password, and is not explicitly tied to any single Tamr instance. For more details, see the documentation for the `Requests library <https://requests.readthedocs.io/en/master/user/advanced/#session-objects>`_.
+
+- Use your username and password to create an instance of ``tamr_client.UsernamePasswordAuth``.
+
+- Use the function ``tamr_client.session.from.auth`` to create a ``Session``.
+
+.. literalinclude:: ../../../tamr_client/examples/get_tamr_version.py
+    :language: python
+    :lines: 1-9
```

Comment:
If the main point of this PR is to use `.. literalinclude` to reference examples, you could keep the doc as `.md` and use [embedded .rst](https://recommonmark.readthedocs.io/en/latest/auto_structify.html#embed-restructuredtext):

```suggestion
    ```eval_rst
    .. literalinclude:: ../../../tamr_client/examples/get_tamr_version.py
    :language: python
    :lines: 1-9
    ```
```

Reply:
Good call. That is much simpler.  I am more familiar with Markdown and prefer it.  Is there any other strong reason for the docs to be in Markdown vs reST?

Reply:
Primarily Markdown for ease-of-use (see https://github.com/Datatamer/tamr-client/blob/master/docs/contributor-guide/adr/0004-documentation-and-docstrings.md#consequences).

The only reason I see to use `.rst` is if you end up writing `.md` that is just a bunch of `eval_rst` blocks with nothing else (e.g. reference docs).